docs/conf.py

```diff
@@ -64,7 +63,7 @@
 
 
 def add_refs_to_docstrings(app, what, name, obj, options, lines):
-    if what == "function" and obj.__name__.startswith("check_"):
+    if what == "function" and obj.__name__.startswith("check_") and "Best Practice: " not in obj.__doc__:
```

Comment:
I thought I had added this a while back, but I guess not. In the case where a manual override of the BP <-> check function linkage is required due to name mismatch (or commonly, if there are multiple checks for a single best practice) this can be overridden directly in the check function docstring.